packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -178,25 +205,42 @@ async fn _run(
                         sent_tasks
                             .try_insert(task_id, sent)
                             .map_err(|_| Error::from(format!(
-                                "Inbound message w/o sent task id {:?}", task_id
+                                "Inbound message with duplicate sent task id {:?}", task_id
                             )))?;
                     }
                     _ => {}
                 }
             }
             outbound = nng_receiver.get_recv_result() => {
+                tracing::trace!("Tried to get outbound {outbound:?}");
```

Comment:
Does this print a large amount? I'm weary of us printing whole messages

Reply:
Yeah, also for debugging purposes, removed it.